### PR TITLE
[modules/postgresql] modules for cluster management in postgresql (debian specific)

### PR DESCRIPTION
**Pull request by arthurzenika**
related to #21293

### CODE REVIEW

File: salt/modules/deb_postgres.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# Import salt libs
+import salt.utils
+
+# Import 3rd-party libs
+import salt.ext.six as six
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'postgres'
```
Comment (kiorky) [inline salt/modules/deb_postgres.py]: the virtualname conflicts with the 'postgres' module, what about postgres_cluster ?

Comment (arthurlogilab) [inline salt/modules/deb_postgres.py]: @kiorky this was deliberate, the command line is then unified for postgres operations. But if this is not common pratice I'm can change that.

Comment (kiorky) [inline salt/modules/deb_postgres.py]: Uhm, i though one of the modules declaring the same __virtualname__ wins the name and then they are not merged together, but i can be wrong here.
Interresting, i'm testing this to verify.

Comment (kiorky) [inline salt/modules/deb_postgres.py]: Indeed, modules are merged, something i did not known about the loader.
So yes, as there are no function no conflict, there is no issue left here.

**Comment by kiorky** [comment]
i'm helping @arthurlogilab for the lint stuff to get the tests going further :)

**Comment by kiorky** [comment]
Et voilà !

**Comment by kiorky** [comment]
@arthurlogilab if #20540 is merged, this will broke the thing with ``__virtualname__``.

**Pull request merged**